pkg/kmetrics/record.go

```diff
@@ -1,131 +0,0 @@
-/*
-Copyright 2021 Google LLC.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package kmetrics
-
-import (
-	"context"
-
-	"go.opencensus.io/stats"
-	"go.opencensus.io/tag"
-	"k8s.io/klog/v2"
-)
-
-var (
-	keyFieldName, _              = tag.NewKey("field_name")
-	keyDeprecatingField, _       = tag.NewKey("deprecating_field")
-	keySimplificationAdoption, _ = tag.NewKey("simplification_field")
-	keyK8sMetadata, _            = tag.NewKey("k8s_metadata_transformer")
-	keyHelmMetrics, _            = tag.NewKey("helm_inflator")
-	keyBaseCount, _              = tag.NewKey("base_source")
-	keyPatchCount, _             = tag.NewKey("patch_field")
-	keyTopTierCount, _           = tag.NewKey("top_tier_field")
-)
-
-// RecordKustomizeFieldCountData records all data relevant to the kustomization's field counts
-func RecordKustomizeFieldCountData(ctx context.Context, fieldCountData *KustomizeFieldMetrics) {
```

Comment:
Is there a reason you moved all this code to the other file? It would probably simplify the diff/review if this was kept the same or refactored separately